src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Used by the TaskList class, and handles reading, writing and deleting from lines in the file.
+ */
+public class Storage {
+
+    private String filepath;
+
+    /**
+     * Carries out operations like reading and writing to a specified database file.
+     * @param filepath
+     */
+    public Storage(String filepath) {
+        this.filepath = filepath;
+        try {
+            File file = new File(this.filepath); // Used for read and write operations to the file
+            file.createNewFile(); // creates a new file if needed
+        } catch (IOException e) {
+            System.out.println("Failed to create database file: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Reads all lines of the file into a List of strings.
+     *
+     * @return List/<String/> that contains all lines of the file
+     */
+    public List<String> readLinesFromFile() {
+        try {
+            return Files.readAllLines(Paths.get(this.filepath));
+        } catch (IOException e) {
+            System.out.println("Failed to read lines from file: " + e.getMessage());
+            return null;
+        }
+    }
+
+    /**
+     * Given a list of strings, writes to a file.
+     * @param stringTaskList {@code List<String>} that is a list of tasks in their string database representation
+     */
+    public void writeLinesToFile(List<String> stringTaskList) {
+        try {
+            File file = new File(this.filepath);
+            file.createNewFile();
+
+            FileWriter fw = new FileWriter(this.filepath);
+
+            for (String x : stringTaskList) {
+                fw.write(x + "\n");
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Failed to write to file: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Returns a List of strings of tasks in db representation.
+     * @return List/<String/> of tasks in db representation
+     */
+    public List<String> load() {
+        List<String> stringTasksList = this.readLinesFromFile();
+        return stringTasksList;
+    }
+
+    /**
+     * Deletes the contents of the database file.
+     */
+    public void clearFile() {
+        try {
+            Files.write(Paths.get(filepath), new byte[0]);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        String filePath = "data/duketest.txt";
+        Storage fileHandler = new Storage(filePath);
+
+        // Clear and init data/duke.txt
+        fileHandler.clearFile();
+
+        List<String> input = new ArrayList<>();
```

Comment:
I've noticed that the input variable could be clearer as a collection of strings by naming it to either `inputs` or `inputStrings` but other than that the rest of the naming for a collection of objects seems good. 👍